bloom.go

```diff
@@ -33,43 +33,17 @@ Similarly, to test if "Love" is in bloom:
 package bloom
 
 import (
-	"hash"
 	"math"
 
 	"github.com/m3db/bitset"
-	stackmurmur3 "github.com/m3db/stackmurmur3"
-
-	"github.com/spaolacci/murmur3"
+	"github.com/twmb/murmur3"
 )
 
-// Hash128 the golang stdlib does not have a Hash128 definition.
-type Hash128 interface {
-	hash.Hash
-	Sum128() (uint64, uint64)
-}
-
 var entropy = []byte{1}[:]
 
-func bloomFilterHashes(hash Hash128, data []byte) [4]uint64 {
-	hash.Reset()
-	hash.Write(data)
-	h1, h2 := hash.Sum128()
-	hash.Write(entropy) // Add entropy
-	h3, h4 := hash.Sum128()
-	return [4]uint64{h1, h2, h3, h4}
-}
-
-// concurrentBloomFilterHashes provides the same hashes as bloomFilterHashes
-// but allows the caller to not have to provide a resuseable hash, this is
-// useful in highly concurrrent call paths that would otherwise have to take
-// a ref to a hash from a shared pool (involving a write lock or channel) or
-// allocate a new hash per operation (expensive for the GC).
 func concurrentBloomFilterHashes(data []byte) [4]uint64 {
-	var hash stackmurmur3.Digest128
-	hash = hash.Write(data)
-	h1, h2 := hash.Sum128()
-	hash = hash.Write(entropy) // Add entropy
-	h3, h4 := hash.Sum128()
+	h1, h2 := murmur3.Sum128(data)
```

Comment:
Isn't this different semantically from the previous code? That is, previous code would be the equivalent of:

```
h1, h2 := murmur3.Sum128(data)

// hash hasn't reset I assume, so the new call is actually on data + entropy)
h3, h4 := murmur3.Sum128(data + entropy)

``

Reply:
yes, I completely misread the code.